packages/terra-tabs/src/clinical-lowlight-theme/CommonTabButton.module.scss

```diff
@@ -0,0 +1,16 @@
+:local {
+  .clinical-lowlight-theme {
+    --terra-tabs-workspace-workspace-button-color: #868a8c;
```

Comment:
if these styles are used in both framework tabs and workspace. then we can keep the names as common-tabs

Reply:
Since package name is "terra-tabs" style variables expect to be prefixed with "--terra-tabs" (causes stylelint err)